example/streamed-list-objects-local/streamedListObjectsLocal.mjs

```diff
@@ -0,0 +1,62 @@
+import { ClientConfiguration, OpenFgaClient, ConsistencyPreference } from "../../dist/index.js";
+
+const apiUrl = process.env.FGA_API_URL || "http://localhost:8080";
+
+async function main() {
+    const client = new OpenFgaClient(new ClientConfiguration({ apiUrl }));
+
+    console.log("Creating temporary store");
+    const { id: storeId } = await client.createStore({ name: "streamed-list-objects-local" });
+
+    const clientWithStore = new OpenFgaClient(new ClientConfiguration({ apiUrl, storeId }));
+
+    const model = {
+        schema_version: "1.1",
+        type_definitions: [
+            { type: "user" },
+            {
+                type: "document",
+                relations: { can_read: { this: {} } },
+                metadata: {
+                    relations: {
+                        can_read: {
+                            directly_related_user_types: [{ type: "user" }]
+                        }
+                    }
+                }
+            }
+        ]
+    };
```

Comment:
Use the [syntax transformer](https://github.com/openfga/language/blob/main/pkg/js/README.md) here:

```suggestion
    const dslString = `
model
  schema 1.1

type user
type document
  relations
    can_read: [user]
`;
   const model = transformer.transformDSLToJSONObject(dslString);
```